Backend/BikesRentalServer.Tests/UsersService/BlockUser.cs

```diff
@@ -0,0 +1,107 @@
+﻿using BikesRentalServer.DataAccess;
+using BikesRentalServer.Models;
+using BikesRentalServer.Services;
+using BikesRentalServer.Tests.Mock;
+using FluentAssertions;
+using Xunit;
+
+namespace BikesRentalServer.Tests.UsersService
+{
+    public class BlockUser
+    {
+        private readonly DatabaseContext _dbContext;
+        private readonly Services.UsersService _usersService;
+
+        public BlockUser()
+        {
+            _dbContext = MockedDbFactory.GetContext();
+            _usersService = new Services.UsersService(_dbContext);
+        }
+
+        [Fact]
+        public void BlockUserThatDoesntExist()
+        {
+            var response = _usersService.BlockUser("");
+
+            response.Status.Should().Be(Status.EntityNotFound);
+            response.Object.Should().Be(null);
+        }
+
+        [Fact]
+        public void BlockUserAlreadyBlocked()
+        {
+            int testId = 100;
```

Comment:
IDE mi podpowiada tu dwie rzeczy:
* użyć `var` zamiast `int` (ale to można zignorować bo to w dużej mierze kwestia moich ustawień)
* użyć `const int`, to akurat można rozważyć. Alternatywnie można nie tworzyć id wcześniej tylko zapisać dodanego usera i wszędzie dalej odwoływać się do `user.Id`

To też się tyczy wszystkich miejsc
@piotrduperas co ty myślisz?